pypeit/spectrographs/ntt_efosc2.py

```diff
@@ -94,6 +96,18 @@ def compound_meta(self, headarr, meta_key):
                 except KeyError:
                     return None
             return decker
+        elif meta_key == 'datasec' or meta_key == 'oscansec':
+            data_x = headarr[0]['HIERARCH ESO DET OUT1 NX'] #valid pixels along X
+            data_y = headarr[0]['HIERARCH ESO DET OUT1 NY'] #valid pixels along Y
+            oscan_y = headarr[0]['HIERARCH ESO DET OUT1 OVSCY'] #Overscan region in Y, no overscan in X
+            pscan_x = headarr[0]['HIERARCH ESO DET OUT1 PRSCX'] #Prescan region in X, no prescan in Y  
+            if meta_key == 'datasec':
+                datasec = '[%s:%s,:%s]' % (pscan_x, pscan_x+data_x, data_y)
+                return datasec
+            else:
+                oscansec = '[:%s,:%s]' % (pscan_x, data_y) # Actually two overscan regions, here I only dealing with the region on x-axis
```

Comment:
One question here: what if I would like to deal with both overscan sections? what should be the string format?

Reply:
Looks like you have decided to avoid overscan sub, right?

Reply:
Yes, removed.

Reply:
Seems I cannot remove this, otherwise, I will get an error saying: `ValueError: zero-size array to reduction operation minimum which has no identity`